### PR TITLE
feat(hostinfo/postfix)

### PR DESCRIPTION
**Pull request by kaustavha**

### CODE REVIEW

**Comment by kaustavha** [comment]
relies on luvit walk from #862

File: hostinfo/postfix.lua [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+    process = line:match('%S+$')
+  })
+  cb()
+end
+
+local ConfigReader = Reader:extend()
+function ConfigReader:_transform(line, cb)
+  local key, value = line:match('^%S+'), line:match('%S+$')
+  local filter = {
+    inet_interfaces = true, -- Postfix Listening On Addresses
```
Comment (rphillips) [inline hostinfo/postfix.lua]: this can be pulled up out of the function

File: hostinfo/postfix.lua [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+--------------------------------------------------------------------------------------------------------------------
+--[[ Postfix ]]--
+local Info = HostInfo:extend()
+function Info:initialize()
+  HostInfo.initialize(self)
+end
+
+function Info:_run(callback)
+
+local function findPostfixProcess(cb)
```
Comment (rphillips) [inline hostinfo/postfix.lua]: looks like the tabbing is slightly off for this function

Comment (kaustavha) [inline hostinfo/postfix.lua]: strange, i wonder why that happened

**Comment by rphillips** [comment]
+1

**Pull request merged**